tests/tools/DEBUG_EdgeMesh.cpp

```diff
@@ -0,0 +1,127 @@
+#include "DEBUG_EdgeMesh.hpp"
+#include <catch2/catch_test_macros.hpp>
+
+namespace wmtk::tests {
+
+// DEBUG_EdgeMesh::DEBUG_EdgeMesh(const EdgeMesh& m)
+//     : EdgeMesh(m)
+// {}
+DEBUG_EdgeMesh::DEBUG_EdgeMesh(EdgeMesh&& m)
+    : EdgeMesh(std::move(m))
+{}
+
+
+bool DEBUG_EdgeMesh::operator==(const DEBUG_EdgeMesh& o) const
+{
+    return static_cast<const DEBUG_EdgeMesh&>(*this) == static_cast<const DEBUG_EdgeMesh&>(o);
```

Comment:
Not sure, in DEBUG_TriMesh it casts to a DEBUG_TriMesh

Reply:
https://github.com/wildmeshing/wildmeshing-toolkit/blob/165270421bd1aa15a2a3ffa5de43e4ddcf21b1c5/tests/tools/DEBUG_TriMesh.cpp#L16

No it doesn't. That would also not make any sense because the cast wouldn't do anything. 

Actually, you created an endless loop here. That also means that this function was never used and never tested. Therefore, throw an exception.